packages/components/src/styles/components/side-nav/header.scss

```diff
@@ -69,21 +69,13 @@
 }
 
 // Dropdown & Button color theming overrides
-.hds-side-nav__dropdown, // deprecated classname
-.hds-side-nav,
-.hds-side-nav-header {
+
+.hds-side-nav {
   .hds-button,
-  .hds-dropdown-toggle-button,
-  .hds-dropdown-toggle-icon {
+  button[class*="hds-dropdown-toggle"] {
```

Comment:
I reduced these to one selector but let me know if you don't like this... @didoo